crates/arrow_sql_gen/src/lib.rs

```diff
@@ -24,3 +24,5 @@ pub mod postgres;
 #[cfg(feature = "sqlite")]
 pub mod sqlite;
 pub mod statement;
+
+pub use sea_query;
```

Comment:
A neat trick - by re-exporting the sea_query crate, we can import it elsewhere without needing to add it to our Cargo.toml dependency.